packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -449,9 +449,16 @@ export class FeatureDevController {
         let session
         try {
             session = await this.sessionStorage.getSession(message.tabID)
+
+            const acceptedFiles = (paths?: { rejected: boolean }[]) =>
+                !paths ? -1 : paths.filter(i => !i.rejected).length
```

Comment:
Why -1?
if file paths are not defined for some reason let's make it 0

Reply:
Looking at the code in here, I was curious, can we don something like 

```typescript
const acceptedFilesLength = (paths?: { rejected: boolean }[]) => (paths || []).filter(i => !i.rejected).length
```